deploy.env

```diff
@@ -0,0 +1,3 @@
+IMAGE_NAME=healthcheck-test
+REGISTRY_URI=595212869906.dkr.ecr.us-east-1.amazonaws.com
+AWS_CLI_REGION=us-east-1
```

Comment:
Put version in here for now.

Reply:
Versioning was removed outright, no need to update this file with that content.